wagtail/admin/apps.py

```diff
@@ -11,6 +11,8 @@ class WagtailAdminAppConfig(AppConfig):
     default_auto_field = "django.db.models.AutoField"
 
     def ready(self):
+        from wagtail.admin.checks import datetime_format_check
```

Comment:
I'm not sure you need this. 

I thought checks run simply by being registered.

Reply:
You're correct @lb- , IDK why i did this. 😅